torchao/experimental/ops/mps/build.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash -eu
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the license found in the
+# LICENSE file in the root directory of this source tree.
+
+cd "$(dirname "$BASH_SOURCE")"
+
+export CMAKE_PREFIX_PATH=$(python -c 'from distutils.sysconfig import get_python_lib; print(get_python_lib())')
+echo "CMAKE_PREFIX_PATH: ${CMAKE_PREFIX_PATH}"
+export CMAKE_OUT=$(python -c "import sys; print(sys.prefix)")/torchao_mps/cmake-out
```

Comment:
not-so-much-of-a-nit: why are you trying to install/cmake-out in python installation directory. Either pick the root directory of the repo or in the local directory and add cmake-out to .gitignore. https://fburl.com/p1n9rex7

Reply:
done!